src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -582,11 +585,12 @@ export default {
 	display: flex;
 	align-items: center;
 	flex: 0 1 auto;
-	// visually balance the menu so it's not
-	// stuck to the scrollbar
-	.action-item {
-		margin-right: 2px;
-	}
+}
+
+/* counter */
+.app-navigation-entry__counter {
+	// Add slightly more space to the right of the counter
+	margin-right: 2px;
 }
```

Comment:
I would remove this, so it is aligned right with the ActionItem. Then also the container is not necessary and the HTML structure doesn't have a double `app-navigation-entry__counter` element. I guess you wanted to center the counter above the ActionItem, but that anyway doesn't really work with a margin-right, since the counter can have different widths. See here (I disabled `margin-right` end it looks good imho):

<img width="1279" alt="AppNavigationItem" src="https://user-images.githubusercontent.com/2496460/116662317-e9a41d00-a995-11eb-9e1e-2e2db54ee7b1.png">

Reply:
However, this wouldn't give a margin between counter and action in case both are present. If we want to have this as well, I would propose to just add
```
&::v-deep .app-navigation-entry__counter {
   margin-right: 2px;
}
```
and leave the rest untouched.